src/pfe/file-watcher/server/src/projects/projectUtil.ts

```diff
@@ -50,13 +50,15 @@ if (process.env.IN_K8) {
 }
 
 const KUBE_NAMESPACE = process.env.KUBE_NAMESPACE || "default";
-const REMOTE_MODE = (process.env.REMOTE_MODE) || false;
+
 
 const containerInfoMap = new Map();
 
 export const containerInfoForceRefreshMap = new Map();
 
-export const LOCAL_WORKSPACE = REMOTE_MODE === "true" ? "/codewind-workspace" : (process.env.NODE_ENV === "test" ? process.env.HOST_WORKSPACE_DIRECTORY : (process.argv[2] ? process.argv[2] : process.env.HOST_WORKSPACE_DIRECTORY));
+
```

Comment:
Can we remove the empty lines?

Reply:
will remove the lines, all the test pass and manual testing is proving its all ok